internal/blockstore/commit_and_query_test.go

```diff
@@ -92,33 +92,10 @@ func TestCommitAndQuery(t *testing.T) {
 		var prevBlockBaseHash, prevBlockHash []byte
 		blockHashes := [][]byte{nil}
 
-		for blockNumber := uint64(1); blockNumber < totalBlocks; blockNumber++ {
-			b := createSampleUserTxBlock(blockNumber, prevBlockBaseHash, prevBlockHash)
-
-			require.NoError(t, env.s.AddSkipListLinks(b))
-			require.NoError(t, env.s.Commit(b))
-
-			height, err := env.s.Height()
-			require.NoError(t, err)
-			require.Equal(t, blockNumber, height)
-
-			blockHeaderBaseBytes, err := proto.Marshal(b.GetHeader().GetBaseHeader())
-			require.NoError(t, err)
-			prevBlockBaseHash, err = crypto.ComputeSHA256Hash(blockHeaderBaseBytes)
-			require.NoError(t, err)
-
-			blockHeaderBytes, err := proto.Marshal(b.GetHeader())
-			require.NoError(t, err)
-			prevBlockHash, err = crypto.ComputeSHA256Hash(blockHeaderBytes)
-			require.NoError(t, err)
-
-			blockHashes = append(blockHashes, prevBlockHash)
-		}
-
-		assertBlocks := func() {
+		assertBlocks := func(startBlockNum, endBlockNum uint64) {
```

Comment:
The changes made to this test do not cover the bug in my environment. That is, I comment out lines 343-345 that seek back to the previous offset, and the test does not fail.

Reply:
In addition, does this test cover the case of reading across multiple chunks?